tests/unit/template_cassdc_test.go

```diff
@@ -551,6 +551,16 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 			Expect(cassdc.Spec.AdditionalSeeds).To(HaveLen(1))
 		})
 
+		It("Setting additionaServiceConfig", func() {
+			options := &helm.Options{
+				KubectlOptions: defaultKubeCtlOptions,
+				ValuesFiles:    []string{"./testdata/additional-service-config-values.yaml"},
+			}
+
+			Expect(renderTemplate(options)).To(Succeed())
+			Expect(cassdc.Spec.AdditionalServiceConfig).ToNot(BeNil())
```

Comment:
Thanks for adding the test. Can you add verification that `AdditionalServiceConfig` matches the expected values? I realize that could get a bit tedious but doing so has a couple benefits. First, it validates and demonstrates how the chart property should be used. Secondly it make it safe to refactor that part of the template in the future should the need arise.

Reply:
New assertions added to verify additional service config values